lib/show.rb

```diff
@@ -0,0 +1,16 @@
+class Show
+  attr_reader :name, :creator, :characters
+
+  def initialize(name, creator, characters)
+    @name = name
+    @creator = creator
+    @characters = characters
+  end
+
+  def total_salary
+    @characters.sum do |character|
+      character.salary
+    end
```

Comment:
good use of `.sum` here. Remember that `do` and `end` can be replaced with curly braces if the line of code between is only a single line, so you can shorten this down to `@characters.sum {|character| character.salary}`